Formula/geocouch.rb

```diff
@@ -137,4 +139,6 @@ def caveats; <<-EOS.undent
     run 'brew info geocouch'.)
     EOS
   end
+
+  test {}
```

Comment:
Actually, I'm not a user of couchdb or geocouch, but I may research it and think of some simple test cases, sure. I'll keep in touch